hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/ozone/container/TestContainerReplication.java

```diff
@@ -134,7 +134,6 @@ public void testContainerReplication() throws Exception {
 
     cluster.shutdownHddsDatanode(keyLocation.getPipeline().getFirstNode());
 
-    waitForReplicaCount(containerID, 2, cluster);
```

Comment:
Why was this removed?  Without this, the test may give false negative: replicate count may be 3 both initially and after re-replication, and we need to distinguish between the two.

Reply:
Because delete thread pool shutdown takes time， when it finishes， the replication manager already have the container replicated. So the wait for 2 replicas randomly fails because of there are already 3 replicas.